griptape/drivers/prompt_model/sagemaker_falcon_prompt_model_driver.py

```diff
@@ -7,17 +8,17 @@
 
 @define
 class SageMakerFalconPromptModelDriver(BasePromptModelDriver):
-    tokenizer: BaseTokenizer = field(
-        default=Factory(
-            lambda self: HuggingFaceTokenizer(
+    _tokenizer: HuggingFaceTokenizer = field(default=None, kw_only=True)
+
+    @property
+    def tokenizer(self) -> HuggingFaceTokenizer:
+        if not self._tokenizer:
```

Comment:
Better to be explicit when checking for `None`. Replace with `if self._tokenizer is None`.